src/main/java/com/epam/reportportal/ws/rabbit/ProjectLogTypesInitializer.java

```diff
@@ -44,33 +44,34 @@
 public class ProjectLogTypesInitializer {
```

Comment:
Let’s give this class a more generic name. I think different functionality could be applied after the project creation event, not only log types init.

Reply:
renamed to ProjectCreatedConsumer